language/polkavm/move-to-polka/src/lib.rs

```diff
@@ -82,12 +86,10 @@ fn link_object_files(
     // let path = Path::new(move_native).to_path_buf();
     // let move_native_path = if move_native_known { &path } else { &out_path };
     // let runtime = get_runtime(move_native_path, &tools)?;
+    let merged_object = out_path.join("merged.o");
```

Comment:
Question: Could we ever see a collision here if we combine something in parallel? Would it make sense to add some randomness to the name `format!("merged_{}.o", random_tag())`?

Reply:
It depends on -o parameter which is output.polkavm by default. Any intermediates will go to automatically created directory which takes name of -o without extension. So modules will be compiled into output/moduleA.o output/moduleB.o and merged will be in output/merged.o. If you use same output for different projects your final output will be overriden anyway, so will be merged.o so I don't see any problems with that. Keeping it stable name its easier to find in debugging is needed later especially with llvm tooling (like section reading).